pkg/util/label.go

```diff
@@ -22,3 +22,15 @@ func MergeLabel(obj *unstructured.Unstructured, labelKey string, labelValue stri
 	workloadLabel[labelKey] = labelValue
 	obj.SetLabels(workloadLabel)
 }
+
+// DedupeAndMergeLabels merges the new labels into exist labels.
+func DedupeAndMergeLabels(existLabel, newLabel map[string]string) map[string]string {
```

Comment:
Perform operations on `existLabel`, the value of `existLabel` will be changed, and therefore no need to return the map value. Or can create a new map object in the function.